signer/xpi/xpi.go

```diff
@@ -278,3 +279,50 @@ func (sig *Signature) String() string {
 	pem.Encode(&buf, &pem.Block{Type: "PKCS7", Bytes: sig.Data})
 	return string(buf.Bytes())
 }
+
+// verifyPKCS7SignatureRoundTrip checks that
+//
+// 1) the signed XPI includes a PKCS7 signature and signature data
+// 2) the signature serializes and deserializes properly
+// 3) the signature cert chain verifies
```

Comment:
you're not verifying chains in the code that follows

Reply:
https://github.com/mozilla-services/autograph/pull/89/files#diff-0bf2ee5528414c7d2ac0e24b14a748daR315 ?

Reply:
you're not providing a `truststore` so chain verification doesn't happen

Reply:
https://github.com/mozilla-services/pkcs7/blob/master/verify.go#L20-L88

Reply:
> you're not providing a truststore so chain verification doesn't happen

that seems like a terrible default

Reply:
updated the docstring